pipeline.wdl

```diff
@@ -21,27 +21,26 @@ version 1.0
 # SOFTWARE.
 
 import "sample.wdl" as sampleWorkflow
-import "tasks/samplesheet.wdl" as samplesheet
+import "structs.wdl" as structs
+import "tasks/biopet.wdl" as biopet
 
 workflow pipeline {
     input {
         Array[File] sampleConfigFiles
         String outputDir
     }
 
-    #  Reading the samples from the sample config files
-    scatter (sampleConfigFile in sampleConfigFiles) {
-        call samplesheet.sampleConfigFileToStruct {
-            input:
-                sampleConfigFile = sampleConfigFile
-        }
+    call biopet.SampleConfigCromwellArrays as configFile {
+      input:
+        inputFiles = sampleConfigFiles,
+        outputPath = "samples.json"
     }
 
-    Array[Sample] samples = flatten(sampleConfigFileToStruct.samples)
+    Root config = read_json(configFile.outputFile)
```

Comment:
I'm guessing read_json always returns a struct? Hence why this Root struct is used, instead of just an Array.